api/v2/serializers/details/__init__.py

```diff
@@ -1,4 +1,6 @@
 from .allocation import AllocationSerializer
+from .allocation_request import AllocationRequestSerializer,\
```

Comment:
Have we decided on an official convention for line-breaking imports yet?  I'm still a fan of using the () syntax to put each on a different line.